### DISCUSSION TITLE
Keywords metadata field is constantly present

### DISCUSSION THREAD

**Discussion by smowtenshi** [estruyf/vscode-front-matter#297] [Q&A] [answered]
Hello.
I created custom `taxonomy.contentTypes` which look like this:
```json
"frontMatter.taxonomy.contentTypes": [
    {
      "name": "default",
      "pageBundle": false,
      "fields": [
        {
          "title": "Title",
          "name": "title",
          "type": "string"
        },
        {
          "title": "Author",
          "name": "author",
          "type": "string",
          "default": "smowtenshi"
        },
        {
          "title": "Publishing date",
          "name": "date",
          "type": "datetime",
          "isPublishDate": true
        },
        {
          "title": "Last modified date",
          "name": "modified",
          "type": "datetime",
          "default": "{{now}}",
          "isModifiedDate": true
        },
        {
          "title": "Slug",
          "name": "slug",
          "type": "string"
        },
        {
          "title": "Summary",
          "name": "summary",
          "type": "string"
        },
        {
          "title": "Is in draft",
          "name": "draft",
          "type": "boolean"
        },
        {
          "title": "Tags",
          "name": "tags",
          "type": "tags"
        },
        {
          "title": "Categories",
          "name": "category",
          "type": "categories"
        }
      ]
    }
  ]
```
yet the Keywords metadata field is still present, but I want to get rid of it.
![image](https://user-images.githubusercontent.com/7839556/159695237-315afd34-8607-4a39-88b1-9622e7f95ac1.png)

Is it possible to disable/remove it or not really?

**Comment by estruyf** [comment, marked as answer]
Hi @smowtenshi at the moment that is not "yet" possible. Keywords are part of the SEO data and will get shown by default. We can for sure create a new setting for this or link it to the #240 issue where we'll allow you to specify what you want to see.

In your case, when you want to hide the keywords field. Do you still want to use the SEO details?

**Reply by estruyf** [reply]
Or we can actually move it here:

![image](https://user-images.githubusercontent.com/2900833/161044176-1f6e1e41-5d52-46b0-a854-28d82bfedf4b.png)

**Reply by smowtenshi** [reply]
> Or we can actually move it here:
> 
I like this approach more, though separate setting to turn it off completely might be a good option as well.

**Reply by estruyf** [reply]
Thank for the feedback, this is now available in the latest beta.

**Reply by AdrianoCahete** [reply]
Can we remove or show separate settings? Like, I need everything BUT the keywords. As far I know, it's not possible to remove just this one, right?

**Reply by estruyf** [reply]
No, this will be there, but it won't do anything if you don't use it. We can think about a setting to disable the keywords field.

**Reply by AdrianoCahete** [reply]
For now, I'm hiding the SEO section enterely, because who will edit does not have knowledge that it cannot be changed. But with that, we'll be hidden the title size and everything else.
It's more a good standard to follow then just a SEO content btw.

**Reply by estruyf** [reply]
Thanks for the feedback @AdrianoCahete. If there are things where the SEO section can be improved, feel free to open an issue.
